### PR TITLE
add unreal theme

### PR DESCRIPTION
**Pull request by ghost**
made a theme inspired by epic games launcher, looks great with frameless window and integrates into epic store
unreal-library

### CODE REVIEW

**Review by redromnon — COMMENTED** [review]
This looks amazing!

File: Unreal/unreal.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.Sidebar {
+  border-right: 3px solid #202020;
+}
+
+[data-testid="MinimizeIcon"] {
+  position: relative;
+  top: -0.25em;
+}
+
+.Sidebar__item {
```
Comment (redromnon) [inline Unreal/unreal.css:50]: I feel this introduces extra space in between the sidebar menu items, compared to the other themes.

File: Unreal/unreal.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  padding: 0.75em 2em;
```
Comment (redromnon) [inline Unreal/unreal.css:51]: I feel this introduces extra space in between the sidebar menu items, compared to the other themes.

Comment (ghost) [reply Unreal/unreal.css:51]: > I feel this introduces extra space in between the sidebar menu items, compared to the other themes.

hi, yes, this is intentional
just tried it without but ultimately all of the padding is necessary to achieve the kind of look that i was going for

although i accounted for this and styled the collapsed sidebar aswell to make sure it looks good

**Comment by redromnon** [comment]
Alright, looks good to me!

**Pull request merged**